### DISCUSSION TITLE
Add content blocks for Markdown on Github

### DISCUSSION THREAD

**Discussion by ghost** [community/community#15327] [Other Feature Feedback, Questions, & Ideas]
Hi all, Community Github!

I have an innovative idea and I would like to know everyone's opinion if it is a good idea or not. I would like if possible to have blocks of content in Markdown on Github. Softwares as Google Docs, Notion, Byword, Confluence, Marktext offer this feature.  I believe this feature can improve ux in Markdown on Github.

#### 1. Feature name:
Content blocks for Markdown on Github.

#### 2.1 User stories:
As a user I would like to view a list of Markdown blocks on Github. This could make my life easier as a developer and writer of technical documents in softwares with Markdown. 

#### 2.2 My idea... with ui/ux:
![image](https://user-images.githubusercontent.com/95204402/165102490-d53d0b40-da2d-4763-a9e9-01913e2ded07.png)

#### 3. Notes:
1. The references I left about the software are not to promote any software company or product, they serve as a bibliographic reference.
2. So, I just want to show you things I've read, researched, tested in this references.
3. Hope to help Github Community.
4. I didn't find anything talking about this feature... I didn't find any requests for this feature.

#### 4. References/Bibliographic reference:
- https://www.google.com/intl/pt-BR/docs/about/
- https://github.com/marktext/marktext
- https://apps.apple.com/us/app/byword/id420212497?mt=12
- https://www.notion.so/
- https://github.com/facebook/lexical

**Comment by ghost** [comment]
#### 1. concept
![](https://aws1.discourse-cdn.com/standard14/uploads/cozic/optimized/2X/9/9e75fc0fc03cfe0ced5ec444abadc309c90e5b28_2_690x388.gif)

#### 2. reference
- https://discourse.joplinapp.org/t/does-joplin-consider-adopting-notions-philosophy/5506
- https://stackoverflow.com/questions/34341808/is-there-a-way-to-add-a-gif-to-a-markdown-file

**Comment by ghost** [comment]
It reminds me of things like Notion, Evernote, Logseq.

**Comment by laughedelic** [comment]
Slash commands are in beta:

- https://docs.github.com/en/issues/tracking-your-work-with-issues/about-slash-commands
- https://github.com/orgs/community/discussions/40299

I think this covers this feature request. It doesn't have all types of "blocks" yet, but I think it's a start.
